ruby/private/binary_wrapper.tpl

```diff
@@ -24,6 +19,7 @@
 #
 
 require 'rbconfig'
+require 'rubygems'
```

Comment:
Why do you need this?

Reply:
Probably because we disable rubygems when we run `ruby` interpreter.

@yugui  I was wondering what was the motivation behind `--disable-rubygems` flag?

Reply:
This is a mess I cleaned up in https://github.com/bazelruby/rules_ruby/pull/38/commits/e93b7b920274300c4582622a5bba30de19408f62